goodtablesio/blueprints/api.py

```diff
@@ -18,14 +22,15 @@ def __init__(self, status_code, message):
         self.status_code = status_code
         self.message = message
 
-    def to_dict(self):
-        return {'message': self.message}
-
 
-@api.app_errorhandler(APIError)
+@api.app_errorhandler(Exception)
 def handle_api_errors(error):
-    response = jsonify(error.to_dict())
-    response.status_code = error.status_code
+    if not isinstance(error, APIError):
+        log.error(repr(error))
```

Comment:
It's wierd I've started with `log.exceptions` but I'm not sure why decided it doesn't work because we outside of the exception stack. 

But now it works :+1: